mfr/core/utils.py

```diff
@@ -76,6 +76,18 @@ def make_renderer(name, metadata, file_path, url, assets_url, export_url):
     :rtype: :class:`mfr.core.extension.BaseRenderer`
     """
     normalized_name = (name and name.lower()) or 'none'
+    if metadata.is_public:
+        try:
+            return driver.DriverManager(
+                namespace='mfr.public_renderers',
+                name=normalized_name,
+                invoke_on_load=True,
+                invoke_args=(metadata, file_path, url, assets_url, export_url),
+            ).driver
+        except:
```

Comment:
Be more explicit on the exception?

Reply:
I think we talked about this. The reason it isn't specific is because any specific exception will be reported below.
If this is new let me know.